it/controllers/FinalTaxCalculationControllerISpec.scala

```diff
@@ -115,7 +115,7 @@ class FinalTaxCalculationControllerISpec extends ComponentSpecBase {
   }
 
   object ExpectedValuesWelsh {
-    val title = toMessages("CY")(s"titlePattern.serviceName.govUk")
+    val title = toMessages("CY")(s"final-tax-overview.individual.heading") + " - " + toMessages("CY")(s"titlePattern.serviceName.govUk") + " - " + messages("base.govUk")
```

Comment:
This can be changed to toMessages("CY")(s"messages(s"htmlTitle", messages("final-tax-overview.individual.heading"))